components/ai_chat/resources/page/components/conversation_list/index.tsx

```diff
@@ -203,6 +203,7 @@ function ConversationList(props: ConversationListProps) {
                       text={latestTurnText}
                       onSubmit={(text) => handleEditSubmit(id, text)}
                       onCancel={() => setEditInputId(null)}
+                      isSubmitDisabled={shouldDisableUserInput}
```

Comment:
Currently we don't expect to handle another request when answer is still generating, so disable the submission here.
For reference, ChatGPT UI also allows to click on edit button and disables the submit edit button when the answer is generating.